product_warranty/readme/USAGE.rst

```diff
@@ -0,0 +1,8 @@
+To use this module, you need to:
+
+#. Go to *Sales > Products > Products (or Product Variants)*
+#. Create a new product (or product variant) or edit an existing one
+   and set 'Warranty Duration' under 'Sales' tab.
+#. If 'Purchase' module is installed, got to
+   *Sales > Products > Products (or Product Variants)*, go to 'Purchase' tab,
+   edit supplier information lines an set the warranty information for each one.
```

Comment:
This option only show if `purchase` addon is installed, right?

Reply:
Yes, that is because of the seller_ids field that is declared in 'product' module, but it is add in the product form view in 'purchase' module.